packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -423,6 +423,27 @@ new lambda.Function(this, 'Function', {
 });
 ```
 
+### Existing Lambda function with CloudFront distribution
```

Comment:
It doesn't feel like this belongs in lambda's README. 

Lambda@Edge is a cloudfront feature and and seems to fit nicely there. Can you revert this file?

Reply:
Done.